screens/CountryDetailScreen.js

```diff
@@ -0,0 +1,164 @@
+import React from "react";
+import {
+  View,
+  Text,
+  StyleSheet,
+  Image,
+  ScrollView,
+  TouchableOpacity,
+  FlatList,
+} from "react-native";
+import { color } from "../assets/color/colors";
+import Icon from "react-native-vector-icons/Ionicons";
+
+const CountryDetailScreen = ({ navigation }) => {
+  const borderCountries = [
+    "Germany",
+    "France",
+    "Netherlands",
+    "Luxembourg",
+    "Germany",
+    "France",
+    "Netherlands",
+    "Luxembourg",
+    "Germany",
+    "France",
+    "Netherlands",
+    "Luxembourg",
+    "Germany",
+    "France",
+    "Netherlands",
+    "Luxembourg",
+  ];
+
+  const renderBorderCountry = ({ item }) => {
+    return (
+      <TouchableOpacity style={styles.borderCountryItem}>
+        <Text style={styles.borderCountryText}>{item}</Text>
+      </TouchableOpacity>
+    );
+  };
+
+  return (
+    <ScrollView contentContainerStyle={styles.container}>
+      <TouchableOpacity
+        onPress={() => navigation.goBack()}
+        style={styles.backButton}
+      >
+        <View style={styles.backContainer}>
+          <Icon name="arrow-back" size={24} color={color.drak_black_000000} />
+          <Text style={styles.backText}>Back</Text>
+        </View>
+      </TouchableOpacity>
+      <View style={styles.content}>
+        <Image
+          source={require("../assets/images/flag.png")}
+          style={styles.image}
+        />
+        <View style={styles.textContainer}>
+          <Text style={[styles.text, styles.textWithMarginTop]}>Belgium</Text>
+          <Text style={styles.text}>NativeName: Berlin</Text>
+          <Text style={styles.text}>Population: 123456789</Text>
+          <Text style={styles.text}>Region: Europe</Text>
+          <Text style={styles.text}>Sub Region: Western Europe</Text>
+          <Text style={styles.text}>Capital: Brussel</Text>
+          <Text style={[styles.text, styles.textWithMarginTop]}>
+            Top level domain: .be
+          </Text>
+          <Text style={styles.text}>Currencies: Euro</Text>
+          <Text style={styles.text}>Languages: English,German</Text>
+          <Text style={[styles.text, styles.textWithMarginTop]}>
+            Border Countries:
+          </Text>
+          <View style={styles.borderCountries}>
+            <FlatList
```

Comment:
In react native Lists should never be nested inside plain ScrollViews and also we dont need FlatList here we can use map as well because max border country length can 14.